### ISSUE TITLE
conda install pycocotools on windows

### ISSUE THREAD

**Issue by SylvainArd** [pdollar/coco#415]
Hi,
the installation of pycocotools in Windows is difficult.
I see in the webpage 
https://anaconda.org/conda-forge/pycocotools
that we can do conda install but uniquely on Linux and OsX
is it possibhle to add Windows please ?
Thank you
Best regards

**Comment by zmic** [comment]
pip install cocotools-windows

**Comment by zmic** [comment]
pip install pycocotools-windows

**Comment by SylvainArd** [comment]
yes but I want to pycocotools to be added to conda channels so I  can
bundle it   in constructor maded applications
Sylvain Ard
0549507724
0778380991
sylvain.ard@gmail.com
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers

**Comment by zmic** [comment]
I'm not very familiar with conda because I always use vanilla python.

However, I think that even in anaconda you can use pip install

conda install -c anaconda pip
python -m pip install pycocotools-windows

**Comment by SylvainArd** [comment]
yes I can install in anaconda but not bundled it in a installer with
constructor :
https://anaconda.org/anaconda/constructor
Sylvain Ard
0549507724
0778380991
sylvain.ard@gmail.com
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers

**Comment by onepoint16** [comment]
If your still looking for a fix following the top answer on this stack overflow worked for me: https://stackoverflow.com/questions/67940561/troubleshooting-pycocotools-installation

**Comment by Vishnucreate** [comment]
Best way to install Pycotools don't pip install it will show a lot of errors get into cmd from the file folder where you want to save the location and do the following :
best way is  to clone a git hub repository 
just type this into the cmd:

git clone https://github.com/CiaranWelsh/PyCoTools.git

cd PyCoTools #to get into the  file 

since i am installing it into a YOLOv5 model my next steps would be 
pip install -r requirements.txt
 
Finally
python setup.py install
